proposed/http-client/http-client-meta.md

```diff
@@ -66,6 +66,14 @@ One could be more granular when defining exception. For example, `TimeOutExcepti
 could be subtypes of `NetworkException`. The chosen approach is not to define such subtypes because
 the exception handling in a consuming library would not be different between those exceptions. 
 
+### Background
+
+The HTTP client PSR has been inspired and created by the [php-http team](https://github.com/orgs/php-http/people). 
+Back in 2015 they created HTTPlug which was an interface for HTTP clients. They wanted an abstraction 
+for third party libraries to rely on so nobody had to rely on Guzzle/Buzz directly. A stable version was tagged 
```

Comment:
maybe "They wanted an abstraction that third party libraries can use to not rely on a specific HTTP client implementation like Guzzle 5, Guzzle 6 or Buzz."

Reply:
Thank you.